### PR TITLE
Make useragent Python-3-compatible, fixes #76

### LINKED ISSUE #76: Invalid syntax for raise with Python 3
I noticed while compiling the package that [this](https://github.com/gwik/geventhttpclient/blob/d66b1d96092e2cc323a906370d2aa792f2345934/src/geventhttpclient/useragent.py#L318) isn't valid in Python 3 and raises a `SyntaxError`.
Maybe you want to use something like [six.reraise](https://pythonhosted.org/six/#six.reraise) for portability?

### PR DESCRIPTION
**Pull request by sbraz**
@rmohr This seems to work fine but I think we should add tests as `useragent` has none yet.

### CODE REVIEW

**Comment by rmohr** [comment]
@sbraz thx. Definitely agree on the necessity to add tests.

**Comment by sbraz** [comment]
I'd add them but I don't really know how the whole module works :)

**Comment by rmohr** [comment]
I will add some. Thanks again. :-)

Louis Sautier <notifications@github.com> schrieb am Di., 19. Juli 2016
16:43:

File: src/geventhttpclient/useragent.py [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
+from six import reraise, print_
 
 import gevent
 try:
     from gevent.dns import DNSError
 except ImportError:
-    class DNSError(StandardError): pass
+    class DNSError(Exception): pass
 
 from url import URL
```
Comment (rmohr) [inline src/geventhttpclient/useragent.py]: from .url import URL

File: src/geventhttpclient/useragent.py [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
 from client import HTTPClient, HTTPClientPool
```
Comment (rmohr) [inline src/geventhttpclient/useragent.py]: from .client import HTTPClient, HTTPClientPool

**Comment by rmohr** [comment]
We also need to replace all basestring checks:
```
s/basestring/six.string_types/g
```

**Comment by rmohr** [comment]
Some first tests are here: https://github.com/gwik/geventhttpclient/pull/77/commits/e0a10b649b71b59cd866f938fb70f430325b448f

**Comment by sbraz** [comment]
Should I pick your commit containing tests into my PR?

**Comment by sbraz** [comment]
Re-pushed with updated code. I also removed an empty line from your test file.

**Comment by sbraz** [comment]
Thanks! Can you make a new release containing these fixes?

**Comment by rmohr** [comment]
@sbraz Thx. I have one follow up pull request #79, after this I think we are ready to release it on pypi.
